climpred/tests/test_graphics.py

```diff
@@ -54,3 +57,49 @@ def test_mpi_pm_plot_bootstrapped_skill_over_leadyear_ds(
     )
     res_ax = plot_bootstrapped_skill_over_leadyear(res)
     assert res_ax is not None
+
+
+@pytest.mark.parametrize('cmap', ['tab10', 'jet'])
+@pytest.mark.parametrize('show_members', [True, False])
+@pytest.mark.parametrize('variable', ['tos', None])
+def test_PerfectModelEnsemble_plot(
+    PM_ds_initialized_1d, PM_ds_control_1d, variable, show_members, cmap
+):
+    """Test PredictionEnsemble.plot()."""
+    pm = PerfectModelEnsemble(PM_ds_initialized_1d)
+    kws = {'cmap': cmap, 'show_members': show_members, 'variable': variable}
+    pm.plot(**kws)
+    pm = pm.add_control(PM_ds_control_1d)
+    pm.plot(**kws)
+
+
+def test_PerfectModelEnsemble_plot_fails_3d(PM_ds_initialized_3d):
```

Comment:
Can you add this for `HindcastEnsemble` too?

Reply:
It is the same code. No changes. That why I thought but sure I can add HE also